tests/test_mlip_calculators.py

```diff
@@ -102,10 +106,13 @@ def test_invalid_device(architecture):
     "architecture, device, kwargs",
     [
         ("alignn", "cpu", {}),
-        ("alignn", "cpu", {"model_path": MODEL_PATH / "v5.27.2024"}),
-        ("alignn", "cpu", {"model_path": MODEL_PATH / "v5.27.2024/best_model.pt"}),
+        ("alignn", "cpu", {"model_path": ALIGNN_PATH}),
+        ("alignn", "cpu", {"model_path": ALIGNN_PATH / "best_model.pt"}),
         ("alignn", "cpu", {"model": "alignnff_wt10"}),
-        ("alignn", "cpu", {"path": MODEL_PATH / "v5.27.2024"}),
+        ("alignn", "cpu", {"path": ALIGNN_PATH} ),
```

Comment:
```suggestion
        ("alignn", "cpu", {"path": ALIGNN_PATH}),
```
I think this is all the pre-commit is complaining about